CITATION.cff

```diff
@@ -0,0 +1,37 @@
+abstract: "marimo is a reactive notebook for Python — runs reproducible experiments, executes as a script, deploys as an app, and versions with git."
+authors:
+  - family-names: Agrawal
+    given-names: Akshay
+    orcid: "https://orcid.org/<insert-number>"
+  - family-names: Scolnick
+    given-names: Myles
+    orcid: "https://orcid.org/0009-0003-4254-9632"
+cff-version: 1.2.0
+date-released: "2023-08-15"
+identifiers:
+  - description: "This is the collection of archived snapshots of all versions of marimo"
+    type: doi
+    value: 10.5281/zenodo.<number>
+  - description: "This is the archived snapshot of version 0.7.2 of marimo"
+    type: doi
+    value: 10.5281/zenodo.<number>
+keywords:
+  - python
+  - research
+  - "data science"
+  - reactive
+  - pipeline
+  - web-app
+  - "data visualization"
+  - "artificial intelligence"
+  - "developer tools"
+  - dataflow
+  - notebooks
+  - dag
+type: software
+url: 'https://marimo.io/'
+license: Apache-2.0
+message: "If you use this software, please cite it using these metadata."
+repository-code: "https://github.com/marimo-team/marimo"
+title: "marimo - an open-source reactive notebook for Python"
+version: 0.7.2
```

Comment:
I thought so too, it could prove tiresome to update it with each release; especially since I notice there being bumps in version numbers/releases/tags weekly (notice akshay bumping the version everytime). It would be an additional thing to do and remember for each version release. Should I remove it?

Reply:
If it's optional, I would remove it